lib/navigation/bottom_navigation_bar.dart

```diff
@@ -22,12 +23,13 @@ class _AppBottomNavigationBarState extends State<AppBottomNavigationBar>
   @override
   void initState() {
     super.initState();
-    tabController = TabController(vsync: this, length: 4);
+    tabController = TabController(vsync: this, length: 5);
```

Comment:
I know it was like this before, but move the initialization of `tabs` above this line and set the length to tabs.length instead of a constant. Safer for the future, avoids someone forgetting to change the value.